migrations/migrations/20180607141908-create-user.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Users', {
```

Comment:
great return! Most people forget this one. Also, you can delete the brackets and the return to make it look more functional.